### DISCUSSION TITLE
Boost EXP just for a single faction.

### DISCUSSION THREAD

**Discussion by veselin-ugrinov** [TrinityCore/TrinityCore#26312] [Q&A] [answered]
Hi, guys.. 
Can you please tell me is it possible to boost the exp just for one of the factions ( in my case- Horde) ?
Currently you can boost all exp for both of the factions, but it's not what i need.

**Comment by ghost** [comment]
I don't know exactly how,. but I imagine that it involves writing custom C++ code to target a specific faction, since all the existing code does not differentiate between main player factions. Maybe you could start off by looking at which parts of the code involves the existing `XP.RATE.` functionality and see if those parts could be modified with custom code.

**Comment by Faq** [comment, marked as answer]
You could also open issue here as feature request https://github.com/TrinityCore/TrinityCoreCustomChanges

**Comment by ghost** [comment]
Sure, that is a fair point too. TrinityCoreCustomChanges is probably a safer bet if you don't want to look up a useful Forum page about custom code.

(minor digression on my part: The custom feature of giving XP boost to only one of the factions seems almost too much of a customized feature to me, but I am a conservative mind type)
